### ISSUE TITLE
Hrefs at the start of new lines highlight in incorrect places

### ISSUE THREAD

**Issue by nathanielnrn** [welpo/tabi#225]
# Bug Report

## Environment

Zola version: 0.17.2
tabi commit: latest

## Expected Behavior
When a link is at the start of a new line only the relevant hyperlinked text should be highlighted when moused over.
For example: 
<img width="590" alt="image" src="https://github.com/welpo/tabi/assets/8882205/8cd3d3d8-0092-4cf0-bf39-9066adeab237">

## Current Behavior
When a hyperlinked text starts at a newline or goes over a new line boundary, some extra space is highlighted that is not part of the text. This seems to be caused by the `::before` attribute.

See the highlight below the `p` in `prompt.`

<img width="783" alt="image" src="https://github.com/welpo/tabi/assets/8882205/1b455558-9bb5-49b8-93a7-c9c4a46771ea">

## Step to Reproduce
Here is some markdown that produces the issue on a new line as part of a `index.md` file

```
Getting programs to to run on FPGAs is _hard_. If you've ever fought your way
through [Vitis documentation](https://google.com).
```

**Comment by welpo** [comment]
Thanks for the report, Nathaniel.

I can reproduce on Chromium and WebKit browsers. It looks fine on Firefox.

Looking into a fix!

**Comment by welpo** [comment]
I was about to give up when I accidentally fixed it with a single line of css for the `::before` element:

```
max-width: 105%;
```

The 105 is not special; any number above 100 does the trick.

I don't know why this works, but it does (on my end at least), so I merged the changes in #226.

Let me know if the issue persists (or if you encounter any other issues)!

Thanks
